source/includes/api/core/_user_groups.md

```diff
@@ -49,3 +49,53 @@ name | string | user_group name
 parent\_id | integer | parent user_group id
 default | boolean | indicates if this user_group is the default group for the programme
 position | integer | position under the parent user_group, used for ordering
+
+### Manage User Groups
+
+An endpoint to create a usergroup for this programme. Only accessible if the key
+has been granted access to users. In order to create a usergroup a name is required.
+This usergroup can be nested underneath another usergroup by passing the parent
+user group id. Images can be uploaded to the usergroup by passing an image url. If
+no position is passed then it will default to the bottom of the hierachy.
```

Comment:
Double-check: does it not end up as a child of the default user group "users"?

Reply:
Just checked and it just ends up at the bottom of the list ie not nested

Reply:
Ok thanks for checking!